src/components/Header.js

```diff
@@ -0,0 +1,38 @@
+import { PropTypes } from 'prop-types';
+import { MdArrowBackIos, MdSettings, MdOutlineMic } from 'react-icons/md';
+import { NavLink } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { modifyBanner } from '../redux/elements';
+import styles from './Header.module.scss';
+
+const Header = props => {
+  const dispactch = useDispatch();
+  const { title } = props;
+  const clickHanlder = () => {
+    dispactch(modifyBanner());
+  };
+  return (
+    <header className={styles.header}>
+      <div className={styles.left}>
```

Comment:
- [ ] Kindly ensure that the _back_ navigation button is not shown/ on the _homepage_. This button performs no function on the _homepage_.

Reply:
Thanks, I will do that